Kitodo/src/main/java/de/sub/goobi/forms/IndexingForm.java

```diff
@@ -110,6 +106,10 @@ public void run() {
 
     private boolean indexingAll = false;
 
+    // -2 = deletion failed; -1 = mapping failed
+    // 1 = mapping successful; 2 = deletion successfull
+    private int indexState = 0;
```

Comment:
Why you are not using a enum for possible values? You describe them already, why not using them? It would even made code more readable as some magic numbers.